Doc/whatsnew/3.12.rst

```diff
@@ -572,6 +572,11 @@ Deprecated
   :exc:`ImportWarning`).
   (Contributed by Brett Cannon in :gh:`65961`.)
 
+* :func:`pkgutil.find_loader` and :func:`pkgutil.get_loader`
+  now raise :exc:`DeprecationWarning`;
+  use :func:`importlib.util.find_spec` instead.
+  (Contributed by Nikita Sobolev in :gh:`97850`.)
```

Comment:
And let's move this down a bit to the "Pending Removal in Python 3.14" section.

Reply:
Thanks, changed!